Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -434,7 +434,17 @@ private IQueryable<BaseItemEntity> PrepareItemQuery(JellyfinDbContext context, I
         dbQuery = dbQuery.AsSingleQuery()
             .Include(e => e.TrailerTypes)
             .Include(e => e.Provider)
-            .Include(e => e.LockedFields);
+            .Include(e => e.LockedFields)
+            .Include(e => e.UserData);
+
+        // if (filter.Recursive)
+        // {
+        //     dbQuery = dbQuery.Include(e => e.DirectChildren)!
+        //         .ThenInclude(e => e.TrailerTypes)
+        //         .Include(e => e.Provider)
+        //         .Include(e => e.LockedFields)
+        //         .Include(e => e.UserData);
+        // }
```

Comment:
If unsued, remove

Reply:
tell that to the original author of emby's code